tests/formdata.js

```diff
@@ -15,7 +15,7 @@ exports.testPost = function () {
   fd.append('text', http.file('hello', 'hello.txt', 'text/plain'))
   fd.append('textField', 'world')
   fd.append('anotherField', '!')
-  var res = http.post('https://httpbin.test.k6.io/post', fd.body(), {
+  var res = http.post('https://httpbin.org/post', fd.body(), {
```

Comment:
Shouldn't we use QuickPizza for these examples, now? 🤔 

For instance, now https://httpbin.org/ looks down (I'm getting 503 Service Temporarily Unavailable).

Reply:
Yes, I opened an issue for it https://github.com/grafana/jslib.k6.io/issues/162. I fixed the main blockers, but I would not fix all of them, as I'm focused right on restoring pipelines after the website migrations. Someone else should help on it.